src/main/java/ch/njol/skript/expressions/ExprName.java

```diff
@@ -125,9 +132,20 @@ String get(final @Nullable Object o) {
 						return null;
 					final ItemMeta m = ((ItemStack) o).getItemMeta();
 					return m == null || !m.hasDisplayName() ? null : m.getDisplayName();
-				// We must implement this differently in 1.14
-				//} else if (o instanceof Inventory) {
-				//	return ((Inventory) o).getName();
+				} else if (o instanceof Inventory) {
+					if (Skript.isRunningMinecraft(1, 14))
+						return ((Inventory) o).getViewers().get(0).getOpenInventory().getTitle();

```

Comment:
I'd return null instead, since it'd be inconsistent otherwise. We must implement inventories differently in Skript or make an expression specifically for inventory names.

Reply:
I do agree this isn't pretty/perfect, but would you be willing to accept this as is, just so we have something that works for the time being?

If not I understand and Ill change it to return null.

Reply:
IMO this is a good enough workaround.

Reply:
I agree with Snow, it won't be a good way even if it supports every way of getting the name, there can be a virtual inventory without a viewer... I think the users should know how it works, this is a Minecraft change.

If this expression is used to get name of an inventory in 1.14, it should give warning in parse time which refers to the new system, also return null.

Reply:
I will be merging this, but without 1.14 support.